build/images/scripts/install_ovs_windows.ps1

```diff
@@ -0,0 +1,131 @@
+Param(
+    [parameter(Mandatory = $false)] [string] $DownloadDir,
+    [parameter(Mandatory = $false)] [string] $DownloadURL,
+    [parameter(Mandatory = $false)] [string] $OVSInstallDir = "C:\openvswitch"
+)
+
+$ErrorActionPreference = "Stop"
+$OVSDownloadURL = "https://github.com/ruicao93/ovs/releases/download/v2.12.9-win64/ovs-2.12.90-win64.zip"
```

Comment:
we should host this in the Antrea repo / website once we have a version that we can use for the release

Reply:
Sure, I will upload it to Antrea official repo before we release the windows version.